app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -18,6 +18,7 @@ class Exchanges::HbxProfilesController < ApplicationController
   #before_action :authorize_for_instance, only: [:edit, :update, :destroy]
   before_action :check_csr_or_hbx_staff, only: [:family_index]
   before_action :find_benefit_sponsorship, only: [:oe_extendable_applications, :oe_extended_applications, :edit_open_enrollment, :extend_open_enrollment, :close_extended_open_enrollment, :edit_fein, :update_fein, :force_publish, :edit_force_publish]
+  before_action :redirect_if_staff_tab_is_disabled, only: [:staff_index]
```

Comment:
Lets make sure we go in the controller spec for hbx_profiles_controller, scheduled_events, and orphans_controller, and make sure we stub the features being enabled. If there isn't a controller spec for the corresponding action that you added a before_action too, probably no need to worry since you have cucumbers.